runtime/doc/syntax.txt

```diff
@@ -4789,10 +4791,10 @@ ModeMsg		'showmode' message (e.g., "-- INSERT --")
 							*hl-MoreMsg*
 MoreMsg		|more-prompt|
 							*hl-NonText*
-NonText		'~' and '@' at the end of the window, characters from
-		'showbreak' and other characters that do not really exist in
-		the text (e.g., ">" displayed when a double-wide character
-		doesn't fit at the end of the line).
+NonText		'@' at the end of the window, characters from 'showbreak' 
+		and other characters that do not really exist in the text 
+		(e.g., ">" displayed when a double-wide character doesn't 
+		fit at the end of the line).
```

Comment:
Sure, but why? There is "nothing to see" there. `EndOfBuffer` is described as an independent thing to `NonText`, and viceversa. Perhaps it would be best to add a note mentioning that `EndOfBuffer` is highlighted the same as `NonText` by default.